src/components/BasketballCourt/index.tsx

```diff
@@ -0,0 +1,42 @@
+import { Stage, Layer } from "react-konva";
+import { Flex } from "@chakra-ui/react";
+import ThreePointArea from "./ThreePointArea";
+import { ReactReduxContext, Provider } from "react-redux";
+import { calcAxisDelta } from "framer-motion/types/projection/geometry/delta-calc";
```

Comment:
Not used anymore?

Reply:
Thanks for reminding. I have deleted the unused import.